homeassistant/components/zeroconf/__init__.py

```diff
@@ -69,7 +75,12 @@ def zeroconf_hass_start(_event):
         Wait till started or otherwise HTTP is not up and running.
         """
         _LOGGER.info("Starting Zeroconf broadcast")
-        zeroconf.register_service(info)
+        try:
+            zeroconf.register_service(info)
+        except NonUniqueNameException:
+            _LOGGER.warning(
+                "Other Home Assistant instance present in the local network"
```

Comment:
It's not just another Home Assistant instance. It's another Home Assistant instance with the same name.

Reply:
Rephrased